website/inertia/pages/events/show/page.tsx

```diff
@@ -69,8 +72,8 @@ export default function EventRegistrationPage({
 
   const fetchRegistrationStatus = async () => {
     try {
-      // const response = await axios.get('/events/' + eventId + '/is-registered')
-      // setIsRegistered(response.data.isRegistered)
+      const response = await axios.get('/events/' + eventId + '/is-registered')
+      setIsRegistered(response.data.isRegistered)
```

Comment:
Couldn't you have used `tuyau` with route names? Genuine question as I am new to both the codebase and the tech stack.